packages/docs/getting-started/running-a-validator.md

```diff
@@ -548,14 +579,30 @@ screen -r -S celo-validator
 
 ### Reference Script
 
-You can use (and modify if you want) this [reference bash script](../../../scripts/run-docker-validator-network.sh) automating all the above steps. It requires Docker and screen.
+If you want to run everything locally, you can use (and modify if you want) this [reference bash script](../../../scripts/run-docker-validator-network.sh) automating all the above steps.
```

Comment:
Does this script work for running your validator, proxy, and attestations service on separate machines?